dist/VIC-schema.json

```diff
@@ -276,7 +276,23 @@
       "format": "email"
     },
     "photo": {
-      "type": "string"
+      "type": "array",
+      "items": {
+        "type": "object",
```

Comment:
I think we can simply use what's in the items object to define this?

Reply:
Yeah, either way will work for us, the front end right now is coded as if we're using an object, not an array. But it's not hard to change.

Reply:
ok i changed it